apps/web/src/components/ProfilePicture/ProfilePicture.tsx

```diff
@@ -69,7 +70,13 @@ export function ProfilePicture({ userRef, ...rest }: Props) {
 
   return (
     <StyledLink href={userProfileLink}>
-      <ValidProfilePicture tokenRef={token} {...rest} />
+      <NftFailureBoundary
+        tokenId={token.dbid}
+        fallback={<RawProfilePicture letter={firstLetter} {...rest} />}
+        loadingFallback={<RawProfilePicture letter={firstLetter} {...rest} />}
+      >
+        <ValidProfilePicture tokenRef={token} {...rest} />
+      </NftFailureBoundary>
```

Comment:
the selected PFP is "broken" on the backend, so this demo shows the fallback working

shows how easy it is to wrap something in the new boundary!

<img width="161" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/9415144f-2c17-49c1-abe5-6a05218bdd90">

Reply:
yoooo this is nice 🧠 🧠 🧠 🧠